transfer_nlp/plugins/config.py

```diff
@@ -62,12 +62,20 @@
 }
 
 
-def register_plugin(clazz):
-    if clazz.__name__ in CLASSES:
-        raise ValueError(f"{clazz.__name__} is already registered to class {CLASSES[clazz.__name__]}. Please select another name")
+def register_plugin(clazz, alias: str=None):
+    if not alias:
+        if clazz.__name__ in CLASSES:
+            raise ValueError(f"{clazz.__name__} is already registered to class {CLASSES[clazz.__name__]}. Please select another name")
```

Comment:
maybe we should adjust these variable names/errors to account for methods as well

Reply:
All mentions of classes are changed for `registrable`, and `CLASSES` becomes `REGISTRY`